### PR TITLE
Drop patches that were upstreamed into condor-9.0.2

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
Same as before -- can you add some greps to make sure the fixes are actually present?

File: Dockerfile [hunk 1]
```diff
@@ -109,18 +109,16 @@ RUN patch -d / -p0 < /tmp/ssh_q.patch
 COPY hosted-ce/overrides/bosco_cluster_xtrace.patch /tmp
 RUN patch -d / -p0 < /tmp/bosco_cluster_xtrace.patch
 
-# HACK: Don't copy over the SSH pub key to the remote side. We set
-# this up with the site out of band.
+# FIXME: Remove this check after a successful build
+# Don't copy the SSH key (HTCONDOR-270)
 COPY hosted-ce/overrides/HTCONDOR-270.skip-key-copy.patch /tmp
```
Comment (matyasselmeci) [inline Dockerfile:114]: Shouldn't you delete this line, since you're deleting the file?

Comment (brianhlin) [reply Dockerfile:114]: Oops yeah, missed it.

**Comment by brianhlin** [comment]
> Same as before -- can you add some greps to make sure the fixes are actually present?

They're already present but I do hope to drop these checks soon.

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**